openrtb2/app.go

```diff
@@ -6,10 +6,10 @@ import (
 	"github.com/prebid/openrtb/v17/adcom1"
 )
 
-// 3.2.14 Object: App
+// Object: App
 //
 // This object should be included if the ad supported content is a non-browser application (typically in mobile) as opposed to a website.
-// A bid request must not contain both an App and a Site object.
+// A request must not contain more than one of a Site, App or DOOH object.
```

Comment:
This sentence makes perfect sense but I believe its wording might be confusing at first glance. I believe a random reader skimming through this file might get the idea that we could have one of each and not one or the other. Can we rephrase?

Maybe:
```
// A request must contain either a Site, or an App or a DOOH object; only one of the three
```
Or something else. IDK.